### PR TITLE
Recreate awesome books app with es6 imports

### PR DESCRIPTION
**Pull request by billionsjoel**
**This PR recreates the awesome books project using es6**

- Use ES6 syntax.
- Divide your code into modules. Save each module in a separate file in the modules dir. Import modules in the index.js file. For this exercise - the more modules the better!
- Keep all functionalities of the app without errors.
- Use luxon module

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+.input {
+  width: 30rem;
+  height: 2rem;
+  border: 2px solid #000;
+  border-radius: 6px;
+  padding: 0.5rem;
+}
+
+.book-title {
+  /* text-transform: uppercase; */
```
Comment (Mihndim2020) [inline index.css]: - [ ] Please kindly remove commented code from your code base, as it makes your code dirty and less readable.

File: modules/UserInterface.js [hunk 2]
```diff
@@ -0,0 +1,147 @@
+import Book from './Book.js';
+import * as luxon from './Date.js';
+
+// import * as luxon from '../node_modules/luxon/src/luxon.js';
```
Comment (Mihndim2020) [inline modules/UserInterface.js]: - [x] Please kindly remove commented code from your code base, as it makes your code dirty and less readable.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi Team, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: Your project is running as expected. 

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by mikethreels — APPROVED** [review]
Hi Billions, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 :
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by HENRYKC24 — COMMENTED** [review]
Hi @Team,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has already been approved by the previous code reviewer.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by nidalaa** [comment]
As in this project description, the information about working on all repos in the group was missing, we need to approve this PR as submitted individually by all students in your group.


Please request a review from your dashboard once again, so the next reviewer can see your request and approve it.
We are sorry for the confusion <3

**Review by arslanbisharat — COMMENTED** [review]
# Approved 👏 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

To Highlight: 
- The Pull Request has a proper title and description. ✔️ 
- You have added a descriptive Readme file. Good Job. ✔️ 
- There is no linter error is present. ✔️ 
- Project has met all the technical aspects you needed for this project. Keep rocking! ✔️ 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of code reviews per project which you can see from your dashboard. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)_

**Pull request merged**